app/javascript/components/enrollment/steps/Identification.js

```diff
@@ -72,6 +73,41 @@ class Identification extends React.Component {
       });
   }
 
+  renderLanguageOption(language, languageIndex) {
+    let fullySupported = language.supported.sms && language.supported.email && language.supported.phone;
+    let languageText = fullySupported ? language.name : language.name + '*';
+    return (
+      <option key={languageIndex} value={language.name}>
+        {languageText}
+      </option>
+    );
+  }
+
+  renderLanguageSupportMessage(selectedLanguage) {
```

Comment:
Yeah I really like this idea... we should emphasize it more as a warning (maybe not italicized but bold, and a "Warning blah blah"